app/controllers/api/admin/green_lanes/themes_controller.rb

```diff
@@ -0,0 +1,33 @@
+module Api
+  module Admin
+    module GreenLanes
+      class ThemesController < AdminController
+        before_action :check_service, :authenticate_user!
+
+        def index
+          render json: serialize(themes.to_a)
+        end
+
+        private
+
+        def themes
+          @themes ||= ::GreenLanes::Theme.order(Sequel.desc(:id))
+        end
+
+        def serialize(*args)
+          Api::Admin::GreenLanes::ThemeSerializer.new(*args).serializable_hash
+        end
+
+        def serialize_errors(theme)
+          Api::Admin::ErrorSerializationService.new(theme).call
+        end
+
```

Comment:
AFAICT this method is unused - I'm surprised rubocop didn't complain about it?

Reply:
removed